ssh/connect.go

```diff
@@ -12,29 +12,19 @@ import (
 	"github.com/unweave/unweave/api/types"
 )
 
-func Connect(ctx context.Context, connectionInfo types.ConnectionInfo, args []string) error {
-	overrideUserKnownHostsFile := false
-	overrideStrictHostKeyChecking := false
-
-	for _, arg := range args {
-		if strings.Contains(arg, "UserKnownHostsFile") {
-			overrideUserKnownHostsFile = true
-		}
-		if strings.Contains(arg, "StrictHostKeyChecking") {
-			overrideStrictHostKeyChecking = true
-		}
-	}
-
-	if !overrideUserKnownHostsFile {
-		args = append(args, "-o", "UserKnownHostsFile=/dev/null")
-	}
-	if !overrideStrictHostKeyChecking {
-		args = append(args, "-o", "StrictHostKeyChecking=no")
+func Connect(ctx context.Context, connectionInfo types.ConnectionInfo, privKeyPath string) error {
+	var sshArgs []string
+
+	// TODO, we want to allow options to override the UserKnownHostsFile and StrictHostKeyChecking
```

Comment:
This function probably needs to be reverted to the way it was. We want to allow overriding any options we set and the ability to pass arbitrary args to the ssh command

Reply:
Reverted